Add Marketing Doc and Update Team Page in Wiki (#186)
jMonkeyEngine/wiki: docs/modules/ROOT/pages/marketingDoc.adoc

= jMonkeyEngine Documentation
:revnumber: 1.0
:revdate: 2025/08/09
:keywords: documentation, marketing, social, media

WIP Marketing Doc

There are currently two ways to opt into having your game or projects featured on JME's official pages:

Opt-in to the Marketing Rotation:
First, each jme user/studio can submit a specific project to be listed in the JME Marketing Rotation, and these projects will be routinely posted in as fair as possible of a rotation.  
To get your project posted about more, you should provide some links to existing websites or pages for your project(s) so that the current JME Marketing Manager has sources from which to pull 
the best images and videos possible.

> If a project has already been posted about alot and reaches a point where new media is not being provided or cannot be easily found, then reposts of existing media will recieve lower priority.

Reviewer: Minor, it should be a lot

> The current rotation can be found here:

Reviewer: Is this "here" as in this page, or is there a link missing?

Author: The link is missing, I need to get a sharable link from a google sheets page I made and copy/paste that in.

I'll alo make sure to do a final spell check on everything shortly.

Most of this came from notes that I was writing in a notepad document over the past week or two, which was probably a bad choice on my end since notepad doesn't have spellcheck.

= jMonkeyEngine Documentation
:revnumber: 1.0
:revdate: 2025/08/09
:keywords: documentation, marketing, social, media

WIP Marketing Doc

There are currently two ways to opt into having your game or projects featured on JME's official pages:

Opt-in to the Marketing Rotation:
First, each jme user/studio can submit a specific project to be listed in the JME Marketing Rotation, and these projects will be routinely posted in as fair as possible of a rotation.  
To get your project posted about more, you should provide some links to existing websites or pages for your project(s) so that the current JME Marketing Manager has sources from which to pull 
the best images and videos possible.
If a project has already been posted about a lot and reaches a point where new media is not being provided or cannot be easily found, then reposts of existing media will recieve lower priority.

The current rotation can be found here:

And you can opt-in by posting to the Monthly Marketing Thread with the following information:

> Project Name:

Reviewer: These sections look a bit flat.

You could use ==== to give them some sectioning. e.g.

====

Project Name:

Where To Link to: 

Best place{s) to find Media:

Extra Notes:

====

Example:

====

Project Name: The Afflicted Forests
Where To Link to: TheAfflictedForests.com
Extra places to find Media: indiedb, twitter, youtube
Extra Notes: coming soon to steam; currently in closed beta; can contact developer for beta keys

====

The current marketing pool can be found here: https://docs.google.com/spreadsheets/d/19ib-riULtnyA7_-heG-rHTKv9L0QF7DOKvgbua9aMpI/edit?usp=sharing

And you can opt-in by posting to the jME Marketing Thread with the following information about your project:

* Project Name
* Where To Link to
* Best place(s) to find Media
* Extra Notes (optional)

===== Submission Example:

**Project Name**: The Afflicted Forests

**Where To Link to**: https://store.steampowered.com/app/972120/The_Afflicted_Forests

**Best places to find Media**: TheAfflictedForests.com, IndieDB, Twitter, YouTube

**Extra Notes**: Coming soon to Steam; currently in closed beta; can contact developer for beta keys

TIP: Be sure to provide links to as many media sources as possible for your project. By provoding more easily accessible media, you can increase the likelihood and frequenecy that the marketing manager will select your content.

=== 2.) Unique Marketing Requests:
You may also submit a request to the jME Marketing Thread providing a curated post with some accompanying images or videos that you want posted. 

> The Marketing Manager still reserves the right to make some edits or deny requests altogether. Submitting too many requests from the same user/studio in the same month may also result in a longer wait time for the posts to be made, especially if many other projects are currently in the queue waiting to be posted.

Reviewer 2: ```suggestion
The Marketing Manager still reserves the right to make some edits or deny requests altogether. 
It is best to submit one complete entry per month to help keep things running smoothly.
```

> TIP: Users are encouraged to make Unique Marketing Requests whenever your project has reached a major milestone that warrants a timely announcement, such as a game that was just released or that has just had a major feature finished. The Marketing Manager may unintentionally overlook these things when routinely selecting content to post from projects that are in the Marketing Pool, so Unique Marketing Requests are the best way to ensure that something important is put in the spotlight and not overlooked.

Reviewer 2: ```suggestion
TIP: Whenever your project hits a big milestone, like a game launch or a major new feature, consider sending a Unique Marketing Request. Since the Marketing Manager is usually picking from the Marketing Pool, things can sometimes slip through the cracks. This is the best way to make sure your big updates get noticed!
```

== Content Posted in jME's Monthly WIP (Work in Progress) Threads

> Any media that users post to any Monthly WIP Thread on the official jME forums (hub.jmonkeyengine.org) may be have their media selected and used for marketing or promotional reasons, unless explicitly requests otherwise (contact the Marketing Manager to make this request). 

Reviewer 2: ```suggestion
Any media posted by users in the Monthly WIP Threads on the official jME forum (hub.jmonkeyengine.org) may be shared on jME’s social media channels, unless users explicitly request otherwise by contacting the Marketing Manager or by writing `noshare` at the end of their post.
```

> So if you post something particuarily cool in the monthly WIP thread, it may be reposted on jME's social media pages without you having to make any requests. However higher priority will be given to users and projects that make direct requests, so you should still make requests when you want to ensure that your content is featured more often.

Reviewer 2: ```suggestion
TIP: Higher priority is given to direct requests, so you should still make requests when you want to ensure that your content is featured more often.
```

== Role of the Marketing Manager

When choosing which projects to post, the Marketing Manager should always strive to be fair in their selections while also prioritizing posts that paint jME in the best light possible. 

The Marketing Manager should be receptive to reasonable criticisms or questioning from users that may feel dissatisfied with the
Marketing Manager's decisions regarding which projects to prioritize. Likewise, jME users should be receptive to feedback and reasonable criticisms from the Marketing Manager when resolving any disputes that may arise. Any unresolvable disputes
should be brought up on the jMonkey hub so that the community can share their opinions and make a democratic judgment in order to resolve the dispute.

In the event that the Marketing Manager role is vacant, someone else is encouraged to volunteer to fill the role. Large gaps in posting are severely frowned upon by
every social media platform, and even an occasional weekly or monthly post is better than nothing and requires very low effort. 

> Login information for all of these platforms is linked to the engine's official project email address, and any future Marketing Managers can obtain this information by contacting a past Marketing Manager or Engine Leader.

Reviewer 2: imo since the  "== Role of the Marketing Manager" section is about our internal management and decision making, it can be omitted by the public wiki
